app/views/projects/show/_experiment.html.erb

```diff
@@ -2,21 +2,21 @@
   <div class="panel panel-default experiment-panel"
        data-id="<%= experiment.id %>">
     <div class="panel-heading">
-
-      <div class="dropdown pull-right">
-        <button class="btn btn-link dropdown-toggle"
-                type="button"
-                id="exActionsMenu-<%= experiment.id %>"
-                data-toggle="dropdown"
-                aria-haspopup="true"
-                aria-expanded="true">
-          <span class="visible-xs-inline"><i class="glyphicon glyphicon-sort"></i></span>
-          <span class="caret"></span>
-        </button>
-        <%= render partial: 'experiments/dropdown_actions.html.erb',
-                            locals: { project: @project, experiment: experiment } %>
-      </div>
-
+      <% if is_user_or_higher_of_project @project %>
```

Comment:
I would change this to actual permissions method, since this is very error prone. Imagine someone someday fixing permissions and would be scratching his head why it doesn't respond.